data/human/human missions.txt

```diff
@@ -2551,7 +2551,7 @@ mission "Terminus exploration"
 				`	"How strong of a threat is the pirate fleet?"`
 				`	"Sorry, I don't have time to help you."`
 					decline
-			`	"Just a couple of interceptors," he says. "We were in a scoutship, with no real weapons to speak of, and none of us on the ship have any combat experience. So, we felt it was safer to run away than to try to fight, even though we were leaving valuable equipment behind."`
+			`	"Just a couple of interceptors," he says. "We were in a Scout, with no real weapons to speak of, and none of us on the ship have any combat experience. So, we felt it was safer to run away than to try to fight, even though we were leaving valuable equipment behind."`
```

Comment:
Is this being changed because the ship's name isn't "scoutship"?
I don't think him saying "scoutship" is necessarily wrong.
Maybe he doesn't know that it's called a "Scout." The longer "scoutship" seems like something people less familiar with it might say rather than the contracted "scout." Perhaps a previous version was even called "Scoutship" and he's not aware of the change.
Maybe he's referring to it as a scoutship, not as the scoutship.
He might not even be talking about the Scout at all. Maybe just another ship that is configured for scouting.

Reply:
It was changed originally because `scoutship` doesn't seem to be an actual word, unlike `scout ship`. Then it was [suggested](https://github.com/endless-sky/endless-sky/pull/10001#discussion_r1571378318) that we should just switch it to Scout.

Reply:
I think I'd rather he say "scout ship" than changing it to the exact model name if we're doing it because "scoutship" isn't in the dictionary.
```suggestion
			`	"Just a couple of interceptors," he says. "We were in a scout ship, with no real weapons to speak of, and none of us on the ship have any combat experience. So, we felt it was safer to run away than to try to fight, even though we were leaving valuable equipment behind."`
```